Python/Product/PythonTools/PythonTools/Intellisense/FuzzyCompletionSet.cs

```diff
@@ -221,6 +223,8 @@ public class FuzzyCompletionSet : CompletionSet {
                 }
                 _filteredCompletions.Filter(IsVisible);
             }
+
+            CommitByDefault = DefaultCommitByDefault;

```

Comment:
I *was* comparing the value against the default to see if it had been changed, before switching to the nullable bool. As this is public API, I see no harm in programmatically exposing the default vs. documentation only.